### DISCUSSION TITLE
Is there a way to change the bslib::page_sidebar() icon from chevron-left to something else like a gear?

### DISCUSSION THREAD

**Discussion by Shelmith-Kariuki** [rstudio/bslib#1115] [Q&A] [answered]
Hi, 

I am working on an app that utilizes the bslib::page_sidebar() function. Is there a way of changing the chevron-left icon to a gear? Please see the two images I have attached. I have edited the svg code directly in the developer tools window to show what we are aiming for.

Below is the code I have used to create the dummy ui. 

```
page_sidebar(
  title = "Example dashboard",
  sidebar = sidebar(
    varSelectInput("var", "Select variable", mtcars)
  ),
  card(
    full_screen = TRUE,
    card_header("My plot"),
    plotOutput("p")
  )
)
```
... and this is the svg code replacement 

`<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" class="bi bi-gear-fill collapse-icon" style="height:;width:;fill:currentColor;vertical-align:-0.125em;" aria-hidden="true" role="img"><path fill-rule="evenodd" d="M9.405 1.05c-.413-1.4-2.397-1.4-2.81 0l-.1.34a1.464 1.464 0 0 1-2.105.872l-.31-.17c-1.283-.698-2.686.705-1.987 1.987l.169.311c.446.82.023 1.841-.872 2.105l-.34.1c-1.4.413-1.4 2.397 0 2.81l.34.1a1.464 1.464 0 0 1 .872 2.105l-.17.31c-.698 1.283.705 2.686 1.987 1.987l.311-.169a1.464 1.464 0 0 1 2.105.872l.1.34c.413 1.4 2.397 1.4 2.81 0l.1-.34a1.464 1.464 0 0 1 2.105-.872l.31.17c1.283.698 2.686-.705 1.987-1.987l-.169-.311a1.464 1.464 0 0 1 .872-2.105l.34-.1c1.4-.413 1.4-2.397 0-2.81l-.34-.1a1.464 1.464 0 0 1-.872-2.105l.17-.31c.698-1.283-.705-2.686-1.987-1.987l-.311.169a1.464 1.464 0 0 1-2.105-.872zM8 10.93a2.929 2.929 0 1 1 0-5.86 2.929 2.929 0 0 1 0 5.858z"></path></svg>`

![bslib icons query1](https://github.com/user-attachments/assets/2d71c6bc-6613-4196-86a0-bf4161fa264d)

![bslib icons query2](https://github.com/user-attachments/assets/442f74b9-6f76-4480-bb41-2288adb7f44e)

**Comment by gadenbuie** [comment, marked as answer]
Thanks for the question Shel! Unfortunately, at the moment, we don't have a straight-forward way to change this icon. We do have an open issue tracking the feature request here: https://github.com/rstudio/bslib/issues/598

**Reply by Shelmith-Kariuki** [reply]
Hi Garrick. Okay, thank you! I support the idea of making the icon type and location flexible.
